lib/middlewares/thinkingListener.js

```diff
@@ -156,8 +156,13 @@ function extractDeviceInformation(req, res, next) {
         return value.attributes;
     }
 
+    function excludedModules(value) {
+        return value.module !== 'AV' && value.module !== 'L1';
```

Comment:
Don't know why you say that. This function is used to exclude those modules from a particular kind of processing not from processing them altogether. They end up receiving their response as they should.